app.js

```diff
@@ -103,7 +113,7 @@ io.on('connection', function(socket){
 
     var term;
     if (process.getuid() == 0) {
```

Comment:
@jarrettgilliam How?

By default, the command is '/bin/login'.  If we use the `if(process.getuid() == 0 || opts.command)` and try to run as root using the default, it would instead set the command to null since opts.command is not defined.  

If you run it as non-root, it will have the same behavior as before and use SSH.